crates/zk-por-cli/src/main.rs

```diff
@@ -68,9 +70,10 @@ impl Execute for Option<ZkPorCommitCommands> {
                 check_non_neg_user(prover_cfg)
             }
 
-            Some(ZkPorCommitCommands::VerifyGlobal { proof_path: global_proof_path }) => {
+            Some(ZkPorCommitCommands::VerifyGlobal { proof_path: global_proof_path, vd_path }) => {
                 let global_proof_path = PathBuf::from_str(&global_proof_path).unwrap();
-                verify_global(global_proof_path, true)
+                let vd_path = PathBuf::from_str(&vd_path).unwrap();
```

Comment:
Do we want to do some error message here to make it more user friendly, or do we want to hardcode the VD path for the verifier?